README.MD

```diff
@@ -24,10 +24,21 @@ This repository contains Studio Notebooks, data, and loading scripts to follow t
 
 The directions below detail how to get started.
 
-## Getting Started
+## How to use this repository
+1. Clone this repository: `git clone git@github.com:datastax/graph-book.git`
+2. With DataStax and DataStax Studio [installed and running](#downloading-datastax-graph-and-datastax-studio), go to the [DataStax Studio Home Page](http://localhost:9091/)
+3. Upload the Chapter notebook you want to use via the Notebook Upload Tool
+4. Follow the directions in the Notebook.
+5. (Optional) Set up all chapter examples at once:
+    1. Import each chapter notebook into DataStax Studio
+    2. Create each graph and graph's schema via the chapter's notebook
+    3. Update the DEFAULT_DSBULK_PATH within each chapter's loading file. (All loading scripts are located in `data/ch_x/ch_x_load.sh`)
```

Comment:
All scripts are written such way, so it's just enough to set `DSBULK_PATH` outside of the scripts and it will be picked up, for example, as `DSBULK_PATH=.... data/ch4/load_ch4.sh`. Or just use `export DSBULK_PATH=....` to set it. Another possibility is to add `dsbulk-1.4...../bin` to the `PATH`, like `export PATH=path_to_dsbulk_bin:$PATH`.

All of this is easier than to modify all scripts.

Similar approach could be used for `DEST_KS` for every script.

Reply:
@alexott I agree with your suggestion. I will open a different issue to implement those changes. Thank you!